src/main/java/Parser.java

```diff
@@ -0,0 +1,124 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import Exception.DukeException;
+
+import javax.lang.model.type.ArrayType;
+
+public class Parser {
+    private TaskList taskList;
+    private Storage storage;
+
+
+    public Parser(TaskList taskList, Storage storage) {
+        this.taskList = taskList;
+        this.storage = storage;
+    }
+
+    /**
+     * Returns a boolean if the user inputs "bye"
+     * @param userInput takes in user input as a String
+     * @return a boolean to indicate whether user wants to quit the application
+     */
+    public boolean determineAction(String userInput) {
```

Comment:
Would this method be clearer to the readers if the method name is named to sound like a boolean? Perhaps a suggestion like 'isExit' would be more suitable?